certora/specs/Delegation.spec

```diff
@@ -216,28 +216,34 @@ rule delegatingWithSigUpdatesVotingPower(env e, DelegationToken.Delegation deleg
 }
 
 // Check that the updated voting power of a delegatee after a transfer is lesser than or equal to the total supply of tokens.
-rule updatedDelegatedVPLTEqTotalSupply(env e, address to, uint256 amount) {
+rule updatedDelegatedVPLTEqTotalSupply(address from, address to) {
+    uint256 balanceOfFromBefore = balanceOf(from);
+    uint256 delegatedVotingPowerDelegateeToBefore = delegatedVotingPower(delegatee(to));
+    uint256 totalSupplyBefore = totalSupply();
+    env e;
+
     require e.msg.value == 0;
-    require e.msg.sender != 0;
+    require e.msg.sender == from;
```

Comment:
Let's put this below when it is first used: just before calling `delegate@withrevert`. We could also explain why those require-statements are safe: it's because tjhey are part of a ghost execution